packages/cli/src/cli/loaders/ejs.ts

```diff
@@ -0,0 +1,217 @@
+import { createLoader } from "./_utils";
+
+const EJS_PLACEHOLDER_REGEX = /<%[\-=]?\s*(.*?)\s*%>/g;
+const LOCALIZABLE_ATTRIBUTES: Record<string, string[]> = {
+  meta: ["content"],
+  img: ["alt"],
+  input: ["placeholder"],
+  a: ["title"],
+};
+const UNLOCALIZABLE_TAGS = ["script", "style"];
+
+// Unique identifier for EJS placeholders to prevent conflicts
+const EJS_PLACEHOLDER_PREFIX = "__EJS_PLACEHOLDER_";
+
+interface EJSPlaceholder {
+  id: string;
+  content: string;
+}
+
+function normalizeTextContent(text: string): string {
+  return text.trim();
+}
+
+export default function createEjsLoader(): ILoader<string, Record<string, any>> {
+  return createLoader({
+    async pull(locale, input) {
+      const result: Record<string, any> = {};
+      const placeholders: EJSPlaceholder[] = [];
+      
+      // Replace EJS expressions with placeholders
+      let processedInput = input.replace(EJS_PLACEHOLDER_REGEX, (match, content) => {
+        const id = `${EJS_PLACEHOLDER_PREFIX}${placeholders.length}`;
+        placeholders.push({ id, content: match });
+        return id;
+      });
+
+      // Parse with JSDOM
+      const dom = new JSDOM(processedInput);
+      const document = dom.window.document;
+
+      const getPath = (node: Node, attribute?: string): string => {
+        const indices: number[] = [];
+        let current = node as ChildNode;
+        let rootParent = "";
+
+        while (current) {
+          const parent = current.parentElement as Element;
+          if (!parent) break;
+
+          if (parent === document.documentElement) {
+            rootParent = current.nodeName.toLowerCase();
+            break;
+          }
+
+          const siblings = Array.from(parent.childNodes).filter(
+            (n) => n.nodeType === 1 || (n.nodeType === 3 && n.textContent?.trim())
+          );
+          const index = siblings.indexOf(current);
+          if (index !== -1) {
+            indices.unshift(index);
+          }
+          current = parent;
+        }
+
+        const basePath = rootParent ? `${rootParent}/${indices.join("/")}` : indices.join("/");
+        return attribute ? `${basePath}#${attribute}` : basePath;
+      };
+
+      const processNode = (node: Node) => {
+        let parent = node.parentElement;
+        while (parent) {
+          if (UNLOCALIZABLE_TAGS.includes(parent.tagName.toLowerCase())) {
+            return;
+          }
+          parent = parent.parentElement;
+        }
+
+        if (node.nodeType === 3) {
+          const text = node.textContent || "";
+          const normalizedText = normalizeTextContent(text);
+          
+          // Check if text contains EJS placeholder
+          if (normalizedText.includes(EJS_PLACEHOLDER_PREFIX)) {
+            // Store original EJS expression
+            const placeholder = placeholders.find(p => normalizedText.includes(p.id));
+            if (placeholder) {
+              result[getPath(node)] = placeholder.content;
+            }
```

Comment:
Why are you removing EJS tags (line 33) and then adding them back here? I found the generated code is the same if I removed both parts. Did I miss anything?

Same in `push()` method below.